modules/gapi/include/opencv2/gapi/stereo.hpp

```diff
@@ -0,0 +1,60 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distereoibution and at http://opencv.org/license.html.
+//
+// Copyright (C) 2021 Intel Corporation
+
+#ifndef OPENCV_GAPI_STEREO_HPP
+#define OPENCV_GAPI_STEREO_HPP
+
+#include <opencv2/gapi/gmat.hpp>
+#include <opencv2/gapi/gscalar.hpp>
+#include <opencv2/gapi/gkernel.hpp>
+
+namespace cv { 
+namespace gapi {
+namespace stereo {
+
+enum OutputFormat {
+    disparity,
+    depth
+};
+
+G_TYPED_KERNEL(GStereoBM, <GMat(GMat, GMat,const int , const int, const OutputFormat, const int, const double, const double)>, "org.opencv.stereoBM") {
+    static GMatDesc outMeta(GMatDesc ly, GMatDesc ry, const int, const int, const OutputFormat, const int ddtype,
```

Comment:
Just wondering what is the `ddtype` ?

Reply:
Its a type of output cv::Mat
`ddtype ` -> `outputType`

Reply:
I meant it just `dtype`

Reply:
`outputType` -> `dtype`